src/hooks/useNotificationTypes.tsx

```diff
@@ -329,41 +332,41 @@ export const notificationTypes: NotificationTypeConfig[] = [
       /**
        * @description Re-use for future Season 3 rewards distribution
        **/
-      // const { dydxAddress } = useAccounts();
```

Comment:
[q] do we want to keep this code around? Looks to be the same as the code below but with a different season.

Also, it feels like we should remove the comment above saying "reuse for season 3" since it's implemented now

Reply:
good point with the comment.

not too sure which code is being referred to, but yea this code is not super elegant. We are basically commenting/uncommenting to trigger/hide the notification. Since there are only 4 seasons planned in total, we thought it was ok.

Reply:
ah i was referring to the commented out block of code from 332-366. i was wondering why we didn't delete that code, especially since the new code seems identical except for which season it refers to.

it sounds like we're indexing on being nimble with how quickly we want to be able to bring back the old code which I think is ok - could we add a comment to explain this? as we start adding a bunch of new people who don't know anything about the codebase (me) I can see commented out code start to get lost in the ether instead of being properly deleted in the future.

non-blocking though, so will stamp PR

Reply:
Ah gotcha. Yea I think it was mainly done for connivence of being able to add it back quickly.  We had the `@description Re-use for future Season 3 rewards distribution` comment for clarification, but it could be a bit more clear.

a better automated solution, would be to have the notification on/off dates configurable in some JSON, I think we thought it would be unnecessary for 4 seasons. It could change if we decide to extend the incentive program longer.